lib/spack/spack/cmd/bootstrap.py

```diff
@@ -15,13 +17,48 @@
 import spack.cmd.common.arguments
 import spack.config
 import spack.main
+import spack.mirror
+import spack.spec
+import spack.stage
 import spack.util.path
 
 description = "manage bootstrap configuration"
 section = "system"
 level = "long"
 
 
+# Tarball to be downloaded if binary packages are requested in a local mirror
+BINARY_TARBALL = 'https://github.com/alalazo/spack-bootstrap-mirrors/releases/download/v0.1-rc.2/bootstrap-buildcache.tar.gz'
```

Comment:
Shouldn't this be configurable? That is, shouldn't the user be able to say which binary bootstrap source they want to create a mirror from?

Reply:
See https://github.com/spack/spack/pull/28556#discussion_r870357141